log/development.log

```diff
@@ -0,0 +1,14 @@
+Started GET "/" for 127.0.0.1 at 2018-03-30 17:18:43 +0300
```

Comment:
This file should not be added to the repo

Reply:
@vikabelous done